includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -364,13 +367,37 @@ protected function get_row_data( $item ) {
 					$course_title = '<strong><a class="row-title" href="' . esc_url( $url ) . '">' . apply_filters( 'the_title', $item->post_title, $item->ID ) . '</a></strong>';
 				}
 
+				// Fetch learners in course.
+				$course_args           = array(
+					'post_id' => $item->ID,
+					'type'    => 'sensei_course_status',
+					'status'  => 'any',
+				);
+				$course_students_count = Sensei_Utils::sensei_check_for_activity( apply_filters( 'sensei_analysis_course_learners', $course_args, $item ) );
```

Comment:
Should we add a doc-block for the filter?

Reply:
Do we even need a filter here? I think we could leave it out.

Reply:
I didn't see on other places where we use check_for_activity that we have doc-block

Reply:
> I didn't see on other places where we use check_for_activity that we have doc-block

I meant we need to add a doc-block for the filter (not for the `check_for_activity` call):
`apply_filters( 'sensei_analysis_course_learners', $course_args, $item )`

You might be right and we don't have doc-blocks in all needed places. I just think it would be better to add them, at least with the new code :)

However, as Donna noticed, the filter here might be not needed at all.

Reply:
Removed [filter](https://github.com/Automattic/sensei/pull/4917/commits/d725a2e0310c234343052289eb02bbb9712a26bf)